src/frontend/app/features/service-catalog/add-service-instance/specify-details-step/specify-details-step.component.ts

```diff
@@ -103,31 +104,24 @@ export class SpecifyDetailsStepComponent implements OnDestroy, AfterContentInit
 
     this.spaceScopeSub = this.servicesService.getSelectedServicePlanAccessibility()
       .pipe(
-      map(o => o.spaceScoped),
-      tap(spaceScope => {
-        if (spaceScope) {
-          this.stepperForm.get('org').disable();
-          this.stepperForm.get('space').disable();
-        } else {
-          this.stepperForm.get('org').enable();
-          this.stepperForm.get('space').enable();
-        }
-      })).subscribe();
+        map(o => o.spaceScoped),
+        tap(spaceScope => {
+          if (spaceScope) {
+            this.stepperForm.get('org').disable();
+            this.stepperForm.get('space').disable();
+          } else {
+            this.stepperForm.get('org').enable();
+            this.stepperForm.get('space').enable();
+          }
+        })).subscribe();
   }
 
   setOrg = (guid) => this.store.dispatch(new SetCreateServiceInstanceOrg(guid));
 
-  initServiceInstances = (paginationKey: string) => getPaginationObservables<APIResource<IServiceInstance>>({
-    store: this.store,
-    action: new GetServiceInstances(this.servicesService.cfGuid, paginationKey),
-    paginationMonitor: this.paginationMonitorFactory.create(
-      paginationKey,
-      entityFactory(serviceInstancesSchemaKey)
-    )
-  }, true)
-    .entities$.pipe(
-    share(),
-    first()
+  initServiceInstances = (paginationKey: string) =>
+    getServiceInstancesInCf(this.servicesService.cfGuid, this.store, this.paginationMonitorFactory).pipe(
```

Comment:
Ah, meant use the servicesService's function. Then you don't have to worry about passing store and paginationMonFac around. Noticed there's a lot of unused extra stuff in service helper, are those, including getServiceInstancesInCf used in other PRs?

Reply:
I'll revisit this PR after the Create/Bind Service instance PRs are merged. They  remove the `servicesService` from the stepper forms, since the wizard can be invoked from different contexts.